content/guides/kubernetes/kapp-gs.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Getting Started with kapp
+parent: Kapp
```

Comment:
Ahh got it. So later we need an intro to kapp to start and then this would be there too. Or we dump it under Carvel. For now it would be standalone especially since it's not packaging really.

Reply:
Yup, we can absolutely have it be top-level and remove this line then if you'd like, and then if you have plans for more kapp content, make a "What is kapp?" top-level page